src/framework/uicomponents/view/dialogview.cpp

```diff
@@ -47,11 +47,10 @@ bool DialogView::isDialog() const
 
 void DialogView::beforeOpen()
 {
-    //! NOTE: Ideally we would change appName in App::run and the following would not be necessary. However, this would
-    //! also change various paths (something we want to avoid outside of major releases).
-#ifndef Q_OS_MAC
-    setTitle(application()->unstable() ? "MuseScore Studio Development" : "MuseScore Studio");
-#endif
+    //! NOTE Set default title
+    if (m_title.isEmpty()) {
+        setTitle(mainWindow()->qWindow()->title());
```

Comment:
I'm not sure if this is desirable, because the main window title follows the score name instead staying at "MuseScore Studio [Development]"

Reply:
you right, thanks! 
fixed